src/extensions/exception-autocapture/index.ts

```diff
@@ -1,11 +1,17 @@
-import { _isArray, _isUndefined, logger, window } from '../../utils'
+import { _isArray, _isObject, _isUndefined, logger, window } from '../../utils'
 import { PostHog } from '../../posthog-core'
 import { DecideResponse, Properties } from '../../types'
 import { ErrorEventArgs, ErrorProperties, errorToProperties, unhandledRejectionToProperties } from './error-conversion'
 import { isPrimitive } from './type-checking'
 
 const EXCEPTION_INGESTION_ENDPOINT = '/e/'
 
+export const extendPostHog = (instance: PostHog, response: DecideResponse) => {
```

Comment:
The place I could see this being called, the decide response isn't passed?

Reply:
Good eyes!